src/components/inputs/radio-input/radio-group.js

```diff
@@ -29,7 +29,7 @@ const Group = props => {
   const optionElements = React.Children.map(props.children, (child, index) => {
     // NOTE: Allowing to intersperse other elements than `Option`.
     if (child && child.type.displayName === Option.displayName) {
-      return React.cloneElement(child, {
+      const option = React.cloneElement(child, {
```

Comment:
Nit: maybe `children` as a name would be apt here.

Reply:
const `clonedChild`?